walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/handlers/OpenIdRequestProcessor.kt

```diff
@@ -48,12 +49,12 @@ class OpenIdRequestProcessor: RequestProcessor {
         if (rawRequest !is OpenIdRawRequest)
             throw UnSupportedProtocolException("Received a raw request of unsupported protocol")
         val presentationRequestContent = rawRequest.mapToPresentationRequestContent()
-        var request = if (rawRequest.requestType == RequestType.ISSUANCE)
+        var request: VerifiedIdRequest<*> = if (rawRequest.requestType == RequestType.ISSUANCE)
             handleIssuanceRequest(presentationRequestContent)
         else
             handlePresentationRequest(presentationRequestContent, rawRequest)
         this.requestProcessors.forEach { extension ->
-            request = extension.parse(rawRequest, request)
+            request = extension.parse(rawRequest, request as VerifiedIdPartialRequest) as VerifiedIdRequest<*>
```

Comment:
This will require re-working since the introduction of the partial verified id requirement, and would probably be best left commented out for now.